pylot/perception/detection/traffic_light_det_operator.py

```diff
@@ -43,22 +43,22 @@ def __init__(self, camera_stream: erdos.ReadStream,
         # Load the model from the model file.
         pylot.utils.set_tf_loglevel(logging.ERROR)
         with self._detection_graph.as_default():
-            od_graph_def = tf.GraphDef()
-            with tf.gfile.GFile(self._flags.traffic_light_det_model_path,
+            od_graph_def = tf.compat.v1.GraphDef()
+            with tf.io.gfile.GFile(self._flags.traffic_light_det_model_path,
```

Comment:
I think this operator should not utilize the v1 API. The traffic light detector runs Faster-RCNN, with our weights so it makes sense to move it to tf2's object detection API and migrate the weights.